config/torch/optim/adagrad.py

```diff
@@ -13,6 +14,9 @@
 
 @dataclass
 class AdagradConf:
+    """For more details on parameteres please refer to the original documentation:
+    https://pytorch.org/docs/stable/optim.html#torch.optim.Adagrad
+    """
```

Comment:
Let's hold on with this change until we finalize the change to configen. This is not urgent and we can do it with configen once it's ready.